main.py

```diff
@@ -1,8 +1,10 @@
 from dotenv import load_dotenv
 import os
-import discord
 import datetime
 from pg2db import *
+import discord
```

Comment:
There is a specific order your imports should be in.  You start with standard library, followed by third party libraries and finally your local application imports.  All of the groups should be separated by a blank line.


Please see https://www.python.org/dev/peps/pep-0008/#imports